responders/src/UI/embc-responder/src/app/shared/components/dialog-components/information-dialog-exit-wizard/information-dialog-exit-wizard.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-information-dialog',
+  templateUrl: './information-dialog-exit-wizard.component.html',
+  styleUrls: ['./information-dialog-exit-wizard.component.scss']
+})
+export class InformationDialogExitWizardComponent implements OnInit {
+  @Output() outputEvent = new EventEmitter<string>();
+  @Input() inputEvent: any;
```

Comment:
'Any' should not be used. Please use appropriate type. Code cleanup is already in progress to remove 'any' from codebase